views/prisonerProfile/partials/prisonerProfileTabs.njk

```diff
@@ -13,13 +13,15 @@
   </h2>
   <ul class="govuk-tabs__list">
     {% for tab in tabs %}
-      {% set fullTabPath = '/prisoner/' + prisonerProfileData.offenderNo + tab.path %}
-      {% set isSelected = ' govuk-tabs__list-item--selected' if currentUrlPath.split('?')[0] === fullTabPath %}
-      <li class="{{'govuk-tabs__list-item' + isSelected }}">
-        <a class="govuk-tabs__tab" href="{{ fullTabPath }}">
-        {{ tab.text }}
-        </a>
-      </li>
+      {% if tab.visible %}
```

Comment:
Is that a sneaky fix for the quicklook tab quirk?

Reply:
No but thanks for reminding me, will have a look at that now.